composer.json

```diff
@@ -18,7 +18,9 @@
         "doctrine/common": "^2.5",
         "symfony/serializer": "^3.4.0 || ^4.0.0",
         "symfony/property-access": "^3.4.0 || ^4.0.0",
-        "symfony/filesystem": "^3.4.0 || ^4.0.0"
+        "symfony/filesystem": "^3.4.0 || ^4.0.0",
+        "ocramius/proxy-manager": "*",
```

Comment:
Generally, it's left to the user to include these packages if they want to use lazy functionality. I would not recommend including them in a vendor package, even less so without a useful version constraint.

Reply:
Hi @alcaeus 👋 , thanks for reviewing.
I didn't see it this way, but that's even better. I'll move them to require-dev and add a note in the doc.